api/candlepin-api-spec.yaml

```diff
@@ -8687,16 +8687,11 @@ components:
       allOf:
         - $ref: '#/components/schemas/TimestampedEntity'
         - type: object
-          required:
-            - name
-            - contentLabel
```

Comment:
Why were the 'required' and 'minLength' attributes removed?

Reply:
I see. I guess this is the problem with applying validation throughout the whole API. You can't control when to validate a DTO on only some endpoints and not others, unless you do manual validation.

If this is a one-off instance, then I guess it's ok to remove the validation annotations from it, but if it needs to be done on multiple DTOs, then I would start thinking whether or not we should leave the DTOValidator.validateConstraints method so that we can do validations manually where we need to, or if we even blanket-applying validation to all endpoints even works for us?

Reply:
That is the issue with applying global validations retroactively. It shows us where we did a poor job with manual validation. If one endpoint requires validation while others don't they should not use the same DTO so that the validation is visible in swagger.

Reply:
Looking at the jenkins report, there is quite a few similar issues. Most of them seem to be the spec test sending an incomplete request. Like creating a [content without uuid](https://github.com/candlepin/candlepin/blob/master/api/candlepin-api-spec.yaml#L8626).